src/main/java/org/mozilla/jss/ssl/SSLAlertEvent.java

```diff
@@ -17,6 +18,7 @@
     int level;
     int description;
 
+    transient JSSEngine engine;
```

Comment:
I'm not sure I understand why `engine` is `transient` here?

Reply:
Well, I wrote this originally in #807... ;-)

`SSLAlertEvent` is an existing class that extends [`EventObject`](https://docs.oracle.com/javase/7/docs/api/java/util/EventObject.html) which in turn implements `Serializable`. Since JSSEngine isn't serializable, `transient` marks it as such here in our `SSLAlertEvent` so that it doesn't try to serialize/deserialize it. In particular, across JVM instances/..., the `JSSEngine` isn't portable (in general, because it depends on NSS state that also isn't portable) and thus can't be serializable.

Reply:
Ah, for context, this is the signature of `this.obj` (from `EventObject.getSource()`) -- [how the parent class is implemented](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/EventObject.java#L44-L47):

```java
    /**
     * The object on which the Event initially occurred.
     */
    protected transient Object source;
```

Reply:
OK, thanks!